content/chainguard/libraries/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Chainguard Libraries"
+linkTitle: "Libraries"
+description: "Libraries for your application development"
```

Comment:
Maybe we can add "software" or "packages" in the description to help with people finding this

Reply:
Done